to.etc.domui/src/main/resources/resources/themes/scss/winter/_floatingWindow.scss

```diff
@@ -72,7 +72,7 @@
 }
 
 .ui-flw-btn-close {
-	padding: 0px;
+	padding: 0px 0px 0px 4px;
```

Comment:
Same for here.

Reply:
looks ugly ;) Title gets sticked to close button otherwise, when title has more width than content.